wordpress-mobile/WordPress-iOS
# Pressing enter in the verticals screen now performs an action.

This is the first PR to fix: https://github.com/wordpress-mobile/WordPress-iOS/issues/12047

Pressing enter in the Site Verticals screen will now find attempt to submit the vertical selection with a perfect match.

![enterInSiteVerticals](https://user-images.githubusercontent.com/1836005/60729338-d13d5580-9f18-11e9-9d24-144914608244.gif)

## Testing:

1. Go to the sites list.
2. Create a new WP.com site.
3. At the verticals selection screen type any input and press enter / tap return.
4. If you have a connection, the form should submit.  If you have no connection, nothing should happen.

Try combining pressing enter with tapping on a cell almost simultaneously - it shouldn't crash.

## Update release notes:
